webapp/components/ContributionForm/index.vue

```diff
@@ -6,8 +6,27 @@
         <no-ssr>
           <hc-editor :users="users" :value="form.content" @input="updateEditorContent" />
         </no-ssr>
+        <ds-space margin-bottom="xxx-large" />
+        <ds-flex class="contribution-form-footer">
+          <ds-flex-item :width="{ base: '10%', sm: '10%', md: '10%', lg: '15%' }" />
+          <ds-flex-item :width="{ base: '80%', sm: '30%', md: '30%', lg: '20%' }">
+            <ds-space margin-bottom="small" />
+            <ds-select
+              model="language"
+              :options="form.languageOptions"
+              icon="globe"
+              :placeholder="form.placeholder"
+              :label="$t('contribution.languageSelectLabel')"
+            />
+          </ds-flex-item>
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T07:21:17Z" title="Friday, June 14th 2019, 9:21:17 am +02:00">Jun 14, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -137,7 +138,10 @@ export default {
       this.$refs.contributionForm.update('content', value)
     },
     returnLocaleName(locale) {
-      if (this.$i18n.locale() === locale.code) {
+      if (
+        (this.contribution && this.contribution.language === locale.code) ||
+        (!this.contribution && this.$i18n.locale() === locale.code)
+      ) {
```


What happens in the else case?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T10:52:34Z" title="Friday, June 14th 2019, 12:52:34 pm +02:00">Jun 14, 2019</time>_

---


good question @Tirokk, I didn't think there was a case where `this.$i8n.locale()` is undefined :thinking: 
what would you expect to happen if there is no locale?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-06-14T11:23:04Z" title="Friday, June 14th 2019, 1:23:04 pm +02:00">Jun 14, 2019</time>_

---


Just use english as the world language @mattwr18 , I think. Or?
If the else case is not possible, we don’t need an ìf` here?!

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-06-14T16:47:24Z" title="Friday, June 14th 2019, 6:47:24 pm +02:00">Jun 14, 2019</time>_

---


Addressed in our latest pairing session @Tirokk... unfortunately, I forgot to add you as a co-author, but I think you know I appreciate you!!